source/core/Date.cpp

```diff
@@ -311,14 +372,7 @@ namespace Vireo {
         localtime_r(&timeVal, &tm);
         _systemLocaleTimeZone = int(tm.tm_gmtoff);
 #else
-        // Issue #218 TODO -  FIXME: if utcTime is non-zero, time zone bias should be based on utcTime passed in
-        // (in GMT secs since epoch), not current time.
-        // Possibly use SystemTimeToTzSpecificLocalTime
-        
-        // flipping the sign of the time zone
-        TIME_ZONE_INFORMATION timeZoneInfo;
-        GetTimeZoneInformation(&timeZoneInfo);
-        _systemLocaleTimeZone = -int((timeZoneInfo.Bias) * kSecondsPerMinute);
+        _systemLocaleTimeZone = GetTimeZoneOffsetSeconds(utcTime);
```

Comment:
Glad you took care of this issue, once the pull request is in, please go ahead and close Issue #218